lib/u3d_core/admin_tools.rb

```diff
@@ -22,6 +22,17 @@
 ## --- END LICENSE BLOCK ---
 module U3dCore
   class AdminTools
+    def self.move_os_file(os, source_path, new_path, dry_run:)
+      if os == :windows
+        source_path.tr!('/', '\\')
+        new_path.tr!('/', '\\')
```

Comment:
Move `U3d::Utils.windows_path` to `U3dCore::Helper` and use it instead? It is a bit better than what you're using currently as it makes sure that specific paths will not cause issues (paths with numbers in IIRC).